sys/shell/src/shell.c

```diff
@@ -449,6 +449,33 @@ shell_register(const char *module_name, const struct shell_cmd *commands)
     return 0;
 }
 
+#if MYNEWT_VAL(SHELL_COMPAT)
+#define SHELL_COMPAT_MODULE_NAME "compat"
+static struct shell_cmd compat_commands[MYNEWT_VAL(SHELL_MAX_COMPAT_COMMANDS)];
+static int num_compat_commands;
```

Comment:
be consistent,  either we always initialize statics or never,  please check coding style for this